utilities/verify.sh

```diff
@@ -10,7 +10,7 @@ if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
     chmod 700 $TRAVIS_BUILD_DIR/signing-tools
     tar xvf $TRAVIS_BUILD_DIR/signing-tools.tar -C $TRAVIS_BUILD_DIR/signing-tools
     # Run verify
-    mvn verify
+    mvn verify -P release
```

Comment:
Without the `-P release` javadoc jars are not generated. If javadoc jars are not generated javadoc building is not triggered. Also this allows us to detect whether there are some problems with the packaging plugins before attempting to deploy the artifacts.